test/spec/components/Group.spec.js

```diff
@@ -43,6 +43,11 @@ describe('<Group>', function() {
     container = TestContainer.get(this);
   });
 
+  beforeEach(() => {
+    const requestAnimationFrameSpy = sinon.fake.returns(cb => cb());
+    global.window.requestAnimationFrame = requestAnimationFrameSpy();
+  });
```

Comment:
The spy is not cleaned up properly, we should use sth like:

```javascript
afterEach(() => global.window.requestAnimationFrame.restore());
```

Reply:
Seems like this doesn't work: https://github.com/bpmn-io/properties-panel/actions/runs/6300214448/job/17102537862?pr=293#step:6:243

Reply:
Indeed, it looks like fakes don't have `restore` even though it's in the types file.

https://sinonjs.org/releases/latest/fakes/